vendor_blueprints/export_card_to_file.py

```diff
@@ -0,0 +1,181 @@
+import json
+import re
+import sys
+import argparse
+import requests
+import urllib.parse
+from pydomo import Domo
+
+
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_ACCOUNT = 201
+EXIT_CODE_BAD_REQUEST = 202
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--email', dest='client_id', required=True)
+    parser.add_argument('--password', dest='password', required=True)
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--secret-key', dest='secret_key', required=True)
+    parser.add_argument('--domo-instance', dest='domo_instance', required=True)
+    parser.add_argument('--filename', dest='filename', required=True)
+    parser.add_argument('--file-type', dest='file_type', required=True)
+    parser.add_argument('--card-id', dest='card_id', required=True)
+    args = parser.parse_args()
+    return args
+
+
+DOMO_INSTANCE = "instance_ID"
+
+def get_access_token(email, password):
+    """
+    Generate Access Token for use with internal content APIs.
+
+    email (str): email address of the user on domo.com
+    password (str): login password of the user domo.com
+    """
+    auth_api = f"https://{DOMO_INSTANCE}.domo.com/api/content/v2/authentication"
+
+    auth_body = json.dumps({
+        "method": "password",
+        "emailAddress": email,
+        "password": password
+    })
+
+    auth_headers = {'Content-Type' : 'application/json'}
+    try:
+        auth_response = requests.post(auth_api, data=auth_body, headers=auth_headers)
+    except Exception as e:
+        print(f"Request error: {e}")
+        sys.exit(EXIT_CODE_BAD_REQUEST)
+
+    if auth_response.json()["success"] is False: # Failed to login
+        print(f"Authentication failed due to reason: {auth_response['reason']}")
```

Comment:
This error handling isn't working correctly. Invalid credentials returns `TypeError: 'Response' object is not subscriptable`